pom.xml

```diff
@@ -66,23 +66,18 @@
 		<dependency>
 			<groupId>org.apache.qpid</groupId>
 			<artifactId>qpid-jms-client</artifactId>
-			<version>0.53.0</version>
+			<version>1.11.0</version>
 		</dependency>
 		<dependency>
-			<groupId>org.apache.geronimo.specs</groupId>
```

Comment:
Check if this is still needed?

Reply:
As far as I can tell we dont have use however even though I highly doubt we will ever need to use a version that is drastically different than the one qpid jms depends on, but yeah just for that possibility, let use the jakara-jms-api version in both the pom files